galley/pkg/config/event/handler.go

```diff
@@ -19,6 +19,22 @@ type Handler interface {
 	Handle(e Event)
 }
 
+// SentinelHandler is a special handler that does nothing with the event. It is useful to avoid
+// nil checks on Handler fields. Specialized operations, such as CombineHandlers recognize SentinelHandlers
+// and elide them when merging.
+func SentinelHandler() Handler {
```

Comment:
I'm not sure I understand the use of `Sentinel` in this context. IIUC sentinel values are used to trigger logical conditions.  This just seems to be an `Empty` or `Dummy` handler.